doc/manual/src/command-ref/env-common.md

```diff
@@ -44,6 +44,12 @@ Most Nix commands interpret the following environment variables:
     The Nix search path can also be extended using the `-I` option to
     many Nix commands, which takes precedence over `NIX_PATH`.
 
+    If `NIX_PATH` is not set or empty, Nix will look up the following locations for a revision of `nixpkgs`:
```

Comment:
This is not correct. It doesn't look for `nixpkgs` - it adds these directories to the search path, regardless of whether they contain `nixpkgs`. In particular, the first and third entries can contain non-nixpkgs channels.

Reply:
```suggestion
    If `NIX_PATH` is not set or empty, Nix will look up the following locations to set the search path:
```

We still have no explanations of the two mechanisms (`name=path` and how contents of a directory on the search path are mapped to `<name>` expressions), but that is out of scope here.
Eventually terms like "search path" should link to their definition, too.

Reply:
Maybe something like:

> If `NIX_PATH` environment variable is not set, Nix will fall back to the following list in impure and unrestricted evaluation mode:
> 
>     1. `$HOME/.nix-defexpr/channels`
>     2. `nixpkgs=/nix/var/nix/profiles/per-user/root/channels/nixpkgs`
>     3. `/nix/var/nix/profiles/per-user/root/channels`